npm-packages/meteor-installer/extract.js

```diff
@@ -3,6 +3,7 @@ const child_process = require('child_process');
 const fs = require('fs');
 const Seven = require('node-7z');
 const { resolve, dirname } = require('path');
+const { isLinux } = require('./config.js');
 const tar = require('tar');
 
 const { isMac } = require('./config.js');
```

Comment:
`isMac` is no longer used on this file. It can be removed then.

Reply:
Thanks for flagging. Fixed.